expr/func_groupbytags.go

```diff
@@ -105,8 +106,9 @@ func (s *FuncGroupByTags) Exec(cache map[Req][]models.Series) ([]models.Series,
 	aggFunc := getCrossSeriesAggFunc(s.aggregator)
 
 	// Now, for each key perform the requested aggregation
+	cons, queryCons := summarizeCons(series)
+
 	for name, groupSeries := range groups {
-		cons, queryCons := summarizeCons(series)
```

Comment:
this seems incorrect. it's possible for input series to have different consolidators defined, or am I missing something?

Reply:
That's true, but I think that's true with many of the aggregation functions and it seems first come first serve. 

In general, I think grouping here would be *very* awkward with mixing consolidators and the user is likely doing something...odd.

Reply:
I think this is more correct, though in the case that the groups are mixed, there is still an arbitrary consolidator chosen. But when 'name' is one of the groupByTags, I could this working better